frontend/express/public/javascripts/pre-login.js

```diff
@@ -11,9 +11,9 @@
 * @param {string} key - key from localization property file
 * @memberof Pre Login
 */
-function showMessage(key) {
+function showMessage(key, prop) {
```

Comment:
this should be backwards compatible with other views that use it without **prop**, would it work in those cases?